astropy/units/cds.py

```diff
@@ -15,6 +15,15 @@
     >>> from astropy.units import cds
     >>> q = 10. * cds.lyr  # doctest: +SKIP
 
+A safer way to use a CDS unit is to start a context manager and perform
+any operation related to these units inside of it, for example:
+
+    >>> with cds.enable():
+    >>>     MOMENTUM_UNIT = u.Unit("GeV/c")
+    >>> particle_momentum = 5 * MOMENTUM_UNIT
```

Comment:
To match the narrative text, this should be indented?
```suggestion
    >>>     particle_momentum = 5 * MOMENTUM_UNIT
```

Reply:
This should be correct as it is, as this is exactly the point: you define `MOMENTUM_UNIT` inside the context manager where the units are enabled and then you can use it together with the rest of the code base.

Reply:
In that case, I propose elaborating on "any operation" to make your point clearer. Thanks!